decidim-proposals/app/commands/decidim/proposals/admin/update_proposal_category.rb

```diff
@@ -0,0 +1,68 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Proposals
+    module Admin
+      #  A command with all the business logic when an admin batch updates proposals category.
+      class UpdateProposalCategory < Rectify::Command
+        # Public: Initializes the command.
+        #
+        # category_id - the category id to update
+        # proposal_ids - the proposals ids to update.
+        def initialize(category_id, proposal_ids)
+          @category = Decidim::Category.find_by id: category_id
+          @proposal_ids = proposal_ids
+          @response = { category_name: "", oks: [], kos: [] }
+        end
+
+        # Executes the command. Broadcasts these events:
+        #
+        # - :update_proposals_category - when everything is ok, returns @response.
+        # - :invalid_category - if the category is blank.
+        # - :invalid_proposal_ids - if the proposal_ids is blank.
+        #
+        # Returns @response hash:
+        #
+        # - :category_name - the translated_name of the category assigned
+        # - :oks - Array of names of the updated proposals
+        # - :kos - Array of names of the proposals not updated because they already had the category assigned
```

Comment:
Is `:oks` and `:kos` a standard thing? I feel like they are prompt to typos, maybe something like `:successful` and `:errored` sounds better to me

Reply:
Ok, I'll change it